irohad/main/impl/ordering_init.cpp

```diff
@@ -46,7 +46,7 @@ namespace iroha {
         log_->error(
             "Ledger don't have peers. Do you set correct genesis block?");
       }
-      auto network_address = ledger_peers.value().front().address;
+      auto network_address = ledger_peers.value().front()->address();
```

Comment:
There's an operator-> in optional, so you may use just `ledger_peers->front()->address()`